.gitignore

```diff
@@ -2,3 +2,5 @@
 Gemfile.lock
 .bundle
 vendor
+.idea
+lib/com
```

Comment:
It isn't very good to ignore vendor or lib/com because when you want to do a clean test, git clean -fd doesn't work correctly.